lms/views/api/sync.py

```diff
@@ -33,13 +34,34 @@ def sync(request):
     grading_student_id = request.parsed_params.get("gradingStudentId")
 
     if group_set_id := request.parsed_params.get("group_set_id"):
-        groupings = grouping_service.get_groups(
-            user=request.user,
-            lti_user=request.lti_user,
-            course=course,
-            grading_student_id=grading_student_id,
-            group_set_id=group_set_id,
-        )
+        course_copy_plugin = request.product.plugin.course_copy
```

Comment:
We only have one /sync view for all LMS's so this only needs to happen here once.

On the other hand, because of that we need to implement the empty plugins for Canvas and D2L.